doc/user/depends.xml

```diff
@@ -561,13 +561,6 @@ int main() { printf("Hello, world!\n"); }
 
       </para>
 
-        <para>
```

Comment:
Review note: this is not removed, it is moved down, to let the discussion of the third arg complete.